src/filefinder.cpp

```diff
@@ -749,6 +751,132 @@ bool FileFinder::IsEasyRpgProject(DirectoryTree const& dir){
 	return(ldb_it != dir.files.end() && lmt_it != dir.files.end());
 }
 
+bool FileFinder::IsRPG2kProjectWithRenames(DirectoryTree const& dir) {
+	return !GetRPG2kProjectWithRenames(dir).Empty();
+}
+
+FileFinder::RPG2KNonStandardFilenameGuesser FileFinder::GetRPG2kProjectWithRenames(DirectoryTree const& dir) {
```

Comment:
Can you move the entire guess/remap logic in a new file? My problem here is that I have a huge PR touching FileFinder in the works and such a huge change is terrible to handle for me :(.

No idea. Make a name up: "filefinder_guesser.cpp/h", "file_guesser.cpp/h", "project_guesser.cpp/.h".

Move ``GetRPG2kProjectWithRenames, GuessAndAddLmuExtension`` and ``RPG2KNonStandardFilenameGuesser, RPG2KFileExtRemap`` to this

``IsRPG2kProjectWithRenames`` can stay as it fits with the rest.

Reply:
Done.